view/rooms/app.js

```diff
@@ -873,6 +882,56 @@ function closeConfirmationModal() {
   );
 }
 
+/**
+ *  Pages IMC Team
+ */
+
+/**
+ * PagingButton was clicked so need to show modal.
+ *
+ * @param  {Object} event - Clicked on toggle event.
+ */
+function pagingConfirmationModal(event) {
+  event.preventDefault();
+  pagingIMCConfirmationModal.setAttribute(
+    'style',
+    'display:block;'
+  );
+  pagingIMCConfirmationText.innerText =
+    'Would you like to Page IMC team for this room Incident?';
```

Comment:
It sounds to me that paging IMC team is for solving the (case) incident. How can we rephrase the message to be more like paging IMC team to solve the technical issue in the room/bots?

Reply:
yeah, its not room incident, it's any bad experience with IMC

Reply:
some ideias:
1- _Would you like to Call IMC team?_
2- _Would you like to Page IMC team?_
3-  _ cry for help?_ 😛 
4- _Are you sure you want to Page IMC team?_
5- _please don't page us it's late._ 😛 

Jokes apart, I'm open to suggestions.

Reply:
`Page IMC team if you experience critical issues/blockers using the IMC room and refreshing the browser doesn't help. `
I think we should emphasise that only using the Page button for critical issues. For non-critical issues, they can create a bugs or ping us instead. 
What do you guys think? @diogo-pessoa  @cassiodias  @ConalCosgrove @BrandonMurnane

Reply:
Is it possible to have the message customisable via room type or env var? For example, we can have a generic default message but if the customised message is defined, it overrides it. 

Otherwise, changing the message may require code changes in future.

Reply:
> Is it possible to have the message customisable via room type or env var? For example, we can have a generic default message but if the customised message is defined, it overrides it.
> 
> Otherwise, changing the message may require code changes in future.

yeah, we can put the String as an EnvVar, I think that's the pattern we use to pass in config to a deployed app without code Change.

Reply:
Changes are done I'll deploy on SandBox next week and see how the EnvVar behaves, then we can customize the wording for the modal.